### PR TITLE
Setup configuration

### PR DESCRIPTION
**Pull request by manuelq12**
Setup Configuration with first test using cucumber

### CODE REVIEW

File: test/features/HelloWorld.feature [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@HelloWorld-Feature
```
Comment (yesid-lopez) [inline test/features/HelloWorld.feature]: why do you need to add this tag?

File: test/features/support/steps.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const { expect } = require('chai');
+const { Given, When, Then } = require("cucumber");
+
+const array = [1,2,3];
+let result = 0;
+
+Given("an Array with items", function() {
```
Comment (yesid-lopez) [inline test/features/support/steps.js:7]: you should define the array with items into this step

File: test/features/support/steps.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+});
+When("the value is not present", function() {
+    result = [1,2,3].indexOf(4);
+});
+Then("the result should be {int}", function(arg1) {
```
Comment (yesid-lopez) [inline test/features/support/steps.js]: can you define a better name than arg1?

File: test/features/HelloWorld.feature (hunk 1 above)
Comment (manuelq12) [reply test/features/HelloWorld.feature]: I saw an example, they placed them in every feature they were going to test, but that is when they want to divide different scenarios in one feature right?

File: test/features/support/steps.js (hunk 2 above)
Comment (manuelq12) [reply test/features/support/steps.js:7]: Do they become work like the "describe"? I'm still not to sure how can i organize correctly the steps and tests

File: test/features/support/steps.js (hunk 3 above)
Comment (manuelq12) [reply test/features/support/steps.js]: Yes!

File: test/features/support/steps.js (hunk 2 above)
Comment (yesid-lopez) [reply test/features/support/steps.js:7]: something like this:
```js
let array;
Given("an Array with items", function() {
     array = [1,2,3];
});
```

File: test/features/HelloWorld.feature (hunk 1 above)
Comment (yesid-lopez) [reply test/features/HelloWorld.feature]: yep, I think that for now, it is not needed

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**